spring-kafka/src/main/java/org/springframework/kafka/retrytopic/EndpointCustomizer.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018-2021 the original author or authors.
```

Comment:
Well, the point of Copyright to identify a lifespan of the class.
I don't think its license can somehow affect `2018` since there was not that class yet.
So, our preference just have it in the current year and then change the last one in the future if we really do some changes to that class.

I may not understand the licensing properly, but for consistency I would like to see a Copyright for new classes like `Copyright 2021 the original...`